### ISSUE TITLE
SAML SSO doesn't work in Mautic v5

### ISSUE THREAD

**Issue by TonyBogdanov** [mautic/mautic#13250]
### Mautic Version

5.0.x series

### PHP version

8.1

### What browsers are you seeing the problem on?

Not relevant

### What happened?

Once you configure the SAML single-sign-on section under Authentication Settings and try to open the login page: `/s/saml/login`, Mautic throws a 500 Internal Server Error and logs the following message:

```
[2024-01-19 14:39:18] mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\ClassNotFoundError: "Attempted to load class "Controller" from namespace "Symfony\Bundle\FrameworkBundle\Controller". Did you forget a "use" statement for another namespace?" at /var/www/localhost/default/vendor/lightsaml/sp-bundle/src/LightSaml/SpBundle/Controller/DefaultController.php line 17 {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\ClassNotFoundError(code: 0): Attempted to load class \"Controller\" from namespace \"Symfony\\Bundle\\FrameworkBundle\\Controller\".\nDid you forget a \"use\" statement for another namespace? at /var/www/localhost/default/vendor/lightsaml/sp-bundle/src/LightSaml/SpBundle/Controller/DefaultController.php:17)"} {"hostname":"b643b00ab17a","pid":46}
```

### How can we reproduce this issue?

Enable SAML SSO.

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by mollux** [comment]
@TonyBogdanov 

this sound like https://github.com/mautic/mautic/issues/12797, can you check if you're on the latest version (5.0.2)?

**Comment by TonyBogdanov** [comment]
@mollux Just tested it an it's not resolved. If you download the 5.0.2 release zip and browse the vendor folder you can see the `DefaultController` still extends `Controller` and not `AbstractController`.

My guess is that composer pulls `lightsaml/sp-bundle` from the original repo rather than the Mautic fork (where the fix is implemented).

**Comment by mollux** [comment]
@TonyBogdanov hah, interesting, that's not correct indeed. I'll further check and see what's wrong

**Comment by mollux** [comment]
@TonyBogdanov I made a PR to address this, see https://github.com/mautic/mautic/pull/13253
This ensures the taball releases pull the correct version, composer based installs are not impacted.

**Comment by escopecz** [comment]
The fix https://github.com/mautic/mautic/pull/13253 will be released in Mautic 5.0.3
